src/kv/MemDB.cc

```diff
@@ -10,6 +10,13 @@
 #include <map>
 #include <string>
 #include <memory>
+#if __has_include(<filesystem>)
```

Comment:
https://github.com/ceph/ceph/pull/36282 proposing to drop MemDB completely, do we still want to make these changes?

Reply:
@neha-ojha am i supposed to wait for the merge of #36282 and rebase my change and retest it? i mean, if this change has been tested, probably we could merge it and rebase #36282 instead. it could save some time.

Reply:
no need to wait on the other PR if this is already tested